libmozdata/bugzilla.py

```diff
@@ -22,6 +23,7 @@ class Bugzilla(Connection):
     ATTACHMENT_API_URL = API_URL + '/attachment'
     TOKEN = config.get('Bugzilla', 'token', '')
     # TOKEN = config.get('Allizgub', 'token', '')
+    BUGZILLA_CHUNK_SIZE = 250
```

Comment:
How did you pick 250?

Reply:
Ryan said 100 is good and 500 (I think it's the value you used to use) should be ok.
So 250 seems to be a good compromise... but not sure of that